### DISCUSSION TITLE
bremsstrahlung cdf data

### DISCUSSION THREAD

**Discussion by aeriforme** [epochpic/epoch#167] [Q&A]
Dear developers, 
I'm interested in understanding some details of the bremsstrahlung implementation in epoch. 
I see that, in order to evaluate the energy of the emitted photon, you sample the probability from the tables in `epoch/epoch3d/src/physics_packages/TABLES/br/`. I understand that these data are processed from the Seltzer-Berger tables used in the Geant4 code (which I found in this Geant4 data files directory: `G4EMLOW7.9.1/brem_SB`). I would like to ask you if you could provide the details of such processing, so that it is clear how you exactly obtained the CDF data from the Geant4 data.

**Comment by keithbennett** [comment]
Since this is a support request rather than a potential bug report, it has been moved to the Q&A discussion section.
